src/test/java/lotto/domain/AnswerTest.java

```diff
@@ -0,0 +1,113 @@
+package lotto.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class AnswerTest {
+
+    @DisplayName("1개의 로또와 당첨 번호를 비교하여 1등 결과를 반환하는 기능")
+    @Test
+    void calculateAnswerFirst() {
+        Lotto lotto = new Lotto(List.of(1, 2, 3, 4, 5, 6));
+        Answer answer = new Answer(List.of(6, 5, 4, 3, 2, 1), 7);
+
+        LottoResult lottoResult = answer.compare(lotto);
+        assertThat(lottoResult.isFirst()).isTrue();
```

Comment:
```suggestion
        assertThat(lottoResult).isEqualTo(LottoResult.FIRST);
```

isFirst() 같은 메소드들이 굳이 필요했을까 생각이 듭니다.

Reply:
사실 이부분도 객체지향적으로 생각해본다면 LottoResult.FIRST도 해당 객체의 필드에 직접 접근한다라고 생각했습니다. 따라서 lottoResult에 isFirst()라는 메시지를 던지는 식으로 했습니다. 다만 제가 고민했던 부분은 isFirst(), isSecond()등등의 메소드가 테스트 코드에서만 쓰이기 때문에 조금 고민을 했습니다. 하지만 Enum이기에 허용해 주었습니다.